templates/battles/choose_team.html

```diff
@@ -12,7 +13,6 @@ <h2>Choose your Pokemons!</h2>
         </div>
         {% if form.errors %}
             <div class="form-error">{{ form.non_field_errors }}</div>
-            <div class="form-error">{{ form.errors }}</div>
```

Comment:
why was this removed?

Reply:
Because I don't have any field related errors for this form.

Reply:
I think is better to maintain this indeed, because if, for some reason, you add some field related errors in the future you'll have to remember to add this again on template. Leaving this implicates in any problem?

Reply:
True. I'm gonna add.